consumer/index.js

```diff
@@ -0,0 +1,43 @@
+const rabbitJs = require('rabbit.js');
```

Comment:
this is a quite old package; probably that's why you had some issues with RabbitMq